README.md

```diff
@@ -26,138 +23,131 @@ Kubernetes is:
-      or extensions, or modify the core Kubernetes code
-    - in the [Kubernetes Developer Guide](docs/devel/README.md)
-    - see also [notes on the API](docs/api.md)
-    - see also the [API object documentation](docs/api-reference/README.md), a
-      detailed description of all fields found in the core API objects
-  - **Walkthroughs and examples**
-    - hands-on introduction and example config files
-    - in the [user guide](http://kubernetes.github.io/docs/user-guide/)
-    - in the [docs/examples directory](examples/)
-  - **Contributions from the Kubernetes community**
-    - in the [docs/contrib directory](contrib/)
-  - **Design documentation and design proposals**
-    - for people who want to understand the design of Kubernetes, and feature proposals
-    - design docs in the [Kubernetes Design Overview](docs/design/README.md) and the [docs/design directory](docs/design/)
-    - proposals in the [docs/proposals directory](docs/proposals/)
-  - **Wiki/FAQ**
-    - in the [wiki](https://github.com/kubernetes/kubernetes/wiki)
-    - troubleshooting information in the [troubleshooting guide](http://kubernetes.io/docs/troubleshooting/)
-
-## Community, discussion, contribution, and support
-
-See which companies are committed to driving quality in Kubernetes on our [community page](http://kubernetes.io/community/).
-
-Do you want to help "shape the evolution of technologies that are container packaged, dynamically scheduled and microservices oriented?"
-
-You should consider joining the [Cloud Native Computing Foundation](https://cncf.io/about). For details about who's involved and how Kubernetes plays a role, read [their announcement](https://cncf.io/news/announcement/2015/07/new-cloud-native-computing-foundation-drive-alignment-among-container).
-
-### Code of conduct
-
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
-
-### Are you ready to add to the discussion?
-
-We have presence on:
+Join us on social media and read our blog:
 
  * [Twitter](https://twitter.com/kubernetesio)
  * [Google+](https://plus.google.com/u/0/b/116512812300813784482/116512812300813784482)
- * [Blogger](http://blog.kubernetes.io/)
-
-You can also view recordings of past events and presentations on our [Media page](http://kubernetes.io/media/).
+ * [Blog](http://blog.kubernetes.io/)
 
-For Q&A, our threads are at:
+Ask questions and help answer them on:
 
  * [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
  * [Slack](http://slack.k8s.io/)
 
-### Want to contribute to Kubernetes?
+Attend our key events:
 
-If you're interested in being a contributor and want to get involved in developing Kubernetes, start in the [Kubernetes Developer Guide](docs/devel/README.md) and also review the [contributor guidelines](CONTRIBUTING.md).
+* [kubecon](http://events.linuxfoundation.org/events/kubecon)
```

Comment:
Hmm. I don't see info about kubecon on kubernetes.io. That would be useful to fix.